timbermill-client/src/main/java/com/datorama/oss/timbermill/common/SQLJetDiskHandler.java

```diff
@@ -0,0 +1,276 @@
+package com.datorama.oss.timbermill.common;
+
+import java.io.*;
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.elasticsearch.action.bulk.BulkRequest;
+import org.elasticsearch.common.io.stream.BytesStreamOutput;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.tmatesoft.sqljet.core.SqlJetException;
+import org.tmatesoft.sqljet.core.SqlJetTransactionMode;
+import org.tmatesoft.sqljet.core.table.ISqlJetCursor;
+import org.tmatesoft.sqljet.core.table.ISqlJetTable;
+import org.tmatesoft.sqljet.core.table.SqlJetDb;
+
+public class SQLJetDiskHandler implements DiskHandler {
+	private static final String DB_NAME = "timbermillJet.db";
+	private static final String FAILED_BULKS_TABLE_NAME = "failed_bulks";
+	private static final String ID = "id";
+	private static final String FAILED_TASK = "failedTask";
+	private static final String CREATE_TIME = "createTime";
+	private static final String INSERT_TIME = "insertTime";
+	private static final String TIMES_FETCHED = "timesFetched";
+	private static final String INSERT_TIME_INDEX = "insertTimeIndex";
+	 @Value("${LOCATION_IN_DISK:}")
+	 private static String locationInDisk;
+
+	public final int fetchLimit = 10;
+	public final long waitingTime;
+
+	private static SqlJetDb db;
+	private static ISqlJetTable table;
+
+	private static final String CREATE_TABLE =
+			"CREATE TABLE IF NOT EXISTS " + FAILED_BULKS_TABLE_NAME + " (" + ID + " TEXT PRIMARY KEY, " + FAILED_TASK + " BLOB NOT NULL, " + CREATE_TIME + " TEXT, " + INSERT_TIME + " LONG, "
+					+ TIMES_FETCHED + " INTEGER)";
+	private static final String createInsertTimeIndexQuery = "CREATE INDEX IF NOT EXISTS " + INSERT_TIME_INDEX + " ON " + FAILED_BULKS_TABLE_NAME + "(" +  INSERT_TIME + ")";
+
+	private static final Logger LOG = LoggerFactory.getLogger(SQLJetDiskHandler.class);
+	//public static final boolean exponentialBackoff = false;
+
+	// initializing database
+	static {
```

Comment:
few questions:
1. not sure using a static block for this kind of init is the best way to go.
2. why not declare this handler as a service and use a postConstruct method? 
@edengal WDYT?